python_ta/transforms/type_inference_visitor.py

```diff
@@ -267,40 +267,27 @@ def lookup_typevar(self, node: NodeNG, name: str):
     ##############################################################################
     # Operation nodes
     ##############################################################################
+    @accept_failable
+    def get_call_signature(self, c) -> TypeResult:
+        if isinstance(c, Type) and isinstance(c.__args__[0], _ForwardRef):
+            c = c.__args__[0]
```

Comment:
In general I'm not a fan of this variable rebinding. Don't use sequenced `if`s here, instead just extract what you want.

Also, since you're changing this anyway, you can add in support for built-in type constructors here (+ tests) fairly easily. I.e., something like `int('3')` should type-check to `int`.